src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1648,7 +1648,7 @@ MulticopterPositionControl::task_main()
 						}
 
 						/* vector of desired yaw direction in XY plane, rotated by PI/2 */
-						math::Vector<3> y_C(-sinf(_att_sp.yaw_body), cosf(_att_sp.yaw_body), 0.0f);
+						math::Vector<3> y_C(-sinf(_yaw), cosf(_yaw), 0.0f);
 
 						if (fabsf(body_z(2)) > SIGMA) {
 							/* desired body_x axis, orthogonal to body_z */
```

Comment:
This comment is also out of sync with the code. body_x is now the part of the actual body yaw that is normal to the desired bedy_z